fe/src/pages/RegistrationView.vue

```diff
@@ -0,0 +1,176 @@
+           
+            <v-btn
+                class="me-4"
+                type="submit"
+            >
+                submit
+            </v-btn>
+        
+            <v-btn @click="handleReset">
+                clear
+            </v-btn>
+        </form>
+    </div>
+    
+  </template>
+  <script setup>
+    import { useField, useForm } from 'vee-validate'
+    import { ref } from 'vue'
+
+    const { handleSubmit, handleReset } = useForm({
+      validationSchema: {
+        firstName (value) {
+          if (value?.length >= 2 && value?.length <= 20) return true
+  
+          return 'First Name needs to be at least 2 characters and not more than 20 characters.'
+        },
+
+        lastName (value) {
+          if (value?.length >= 2 && value?.length <= 20) return true
+  
+          return 'Last Name needs to be at least 2 characters and not more than 20 characters.'
+        },
+
+        phone (value) {
+          if (value?.length > 9 && /[0-9-]+/.test(value)) return true
+  
+          return 'Phone number needs to be at least 9 digits.'
+        },
+        email (value) {
+          if (/^[a-z.-]+@[a-z.-]+\.[a-z]+$/i.test(value)) return true
+  
+          return 'Must be a valid e-mail.'
+        },
+        username (value) {
+          if (value?.length >= 5) return true
+  
+          return 'Must be a valid Username.'
+        },
+        password (value) {
+          if (value?.length >= 8 && /^(?=.*[A-Za-z])(?=.*\d).*$/.test(value)) return true
+  
+          return 'Must be a valid password.'
+        },
+        profileImg (value) {
+          if (value != null) return true
+  
+          return 'Please Attach a Photo.'
+        },
+        dtiImg (value) {
+          if (value != null && showHostInput.value == true) return true
+          if(showHostInput.value == false) return true
+          return 'Please Attach a Photo.'
+        },
+      },
+    })
+    
+    const firstName = useField('firstName')
+    const lastName = useField('lastName')
+    const phone = useField('phone')
+    const email = useField('email')
+    const username = useField('username')
+    const password = useField('password')
+    const profileImg = useField('profileImg')
+    const dtiImg = useField('dtiImg')
+    const showHostInput = ref(false);
+    const userType = ref('client');
+    const show1 = ref(false);
+
+
+    const handleUserType = () =>{
+        if(userType.value == 'host'){
+            showHostInput.value = true;
+        }
+        if(userType.value == 'client'){
+            showHostInput.value = false;
+        }
+    }
+  
+    const submit = handleSubmit(values => {
+      alert(JSON.stringify(values, null, 2))
+      
+    })
+  </script>
+
+<style scope>
+    .form-container {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 100vh;
+    }
+    .form-content {
+        max-width: 400px;
+        width: 100%;
+    }
+</style>
```

Comment:
add new line below

Reply:
Deleted this view as it was not important to the component 🙇